test/libsolidity/syntaxTests/controlFlow/storageReturn/unimplemented_library.sol

```diff
@@ -1,4 +1,12 @@
+// This used to work pre-0.6.0.
 library L {
     function f() public returns(uint[] storage);
     function g() public returns(uint[] storage s);
 }
+abstract library T {
+    function f() public returns(uint[] storage);
+    function g() public returns(uint[] storage s);
+}
+// ----
+// TypeError: (32-145): Contract "L" should be marked as abstract.
```

Comment:
Not the best error message...

Reply:
Should be easy to change this to ``library functions must be implemented`` or ``Abstract libraries are not (yet) supported``? If that isn't easy, then it's probably not too bad, though...

Reply:
I know, that's what @axic wanted ;)

Reply:
Yeah, but I merely meant that the error message probably shouldn't suggest to mark it "abstract", if we don't allow doing that :-).